highfidelity/hifi
# Move Toybox Domain to Production s3

This PR collects all of the toybox scripts into unpublishedScripts/domainContent, collects all assets and scripts and changes references to the production version of our s3 bucket so things will be cached.

Due to access restrictions this will need to be QA'd by Hifi employees.

To test

1. go to demo
2. try the toys
3. press the hidden reset on the arcade machine
4. go to another domain.
5. come back (things should load quickly from cache
